requirements.txt

```diff
@@ -2,7 +2,7 @@
 
 BeautifulSoup4
 werkzeug<1.0 # This can be removed when switching from Flask-OAuthlib
-Flask>=0.10
+Flask>=1.1.0
```

Comment:
I'm afraid that updating Flask version creates bugs in other parts of the code, see the error in the tests! 😮

Reply:
😨 Oh no! Hm, all is well when Flask resolves to version 1.0.2 it seems. At least that's what my local has.

Reply:
Another 🥶  ??

Reply:
Let's really try to avoid it. The incompatibility may not be widespread: what we have so far is Validators for Modals. Thats only used in a few places. And possibly correctly in some? 😊

Reply:
It appears the problem is somewhat indirect: the final resloved Flask version appears to have been 1.1.2 for some time in the GitHub Actions test env. The same goes for the prod env. Anyone who can spot which package is actually responsible?

Reply:
> 😨 Oh no! Hm, all is well when Flask resolves to version 1.0.2 it seems. At least that's what my local has.

To update - my local with 1.0.2 actually does fail a pytest from the top with the same error as your 1.0.3. I have apparently been running only selected test locally for a bit.

Reply:
Gee, this is sad on a Friday afternoon 😭

Reply:
Yeah; @mikaell if you are ok with it we’ll fix the WTF modal issue instead (#1880) but it will likely not happen today. You can likely navigate around it for now by explicitly installing the version you want. Thank you for pointing it out! 👍🏻

Reply:
Sure. I'll halt this PR. But maybe there should be documented workarounds somewhere? If not _required_ versions, maybe _recommended_. It took me a good while to find :) (Was trying to deploy another thing and got stuck on this)

Reply:
Yeah, perhaps, but the thing is that since it doesn’t work with the older versions any more, but also not the latest (since like very recently - can’t be more than two weeks or so) we shall end up freezing the version instead. Which is going to be ok today but cause all kinds of compatibility issues between different components and install/deploy problems after some time. So we’d prefer to remove at least the code issues with conflict with the newer versions before we can require a modern version. And if we can manage it wouldn’t hurt fixing those that broke old versions, just in case other components are blocking install of the latest version on some system. But that is not so important. Pedagogical wasn’t it! 😆

Another way of saying it is that if you feel you can get this PR to work against tests, you are welcome to fix it. One way of doing that is by changing that modal verification code and so on as in #1880, but others are welcome as long as it does not involve explicitly stating an exact version of the dependencies.